docs/quickstarts.rst

```diff
@@ -29,26 +29,23 @@ Once generated, run ``cp .env_template .env`` in the fv3net root and edit the ``
 Cloud Authentication
 --------------------
 
-The ``fv3net`` project currently utilizes Google Cloud to deploy workflow items to services such as Kubernetes and Dataflow.  Authentication requires an installation of `Google Cloud SDK <https://cloud.google.com/sdk/docs/install>`_.
+The ``fv3net`` project currently utilizes Google Cloud to deploy workflow items to services such as Kubernetes and Dataflow.
+Authentication requires an installation of `Google Cloud SDK <https://cloud.google.com/sdk/docs/install>`_.
 
-Authentication obtained via ``gcloud auth login`` does not work well with secrets management and is not used by many APIs. Service account key-based authentication works much better, because the service account key is a single file that can be deployed in a variety of contexts (K8s cluter, VM, etc) (`Google docs on service accounts <https://cloud.google.com/iam/docs/service-accounts>`_). Many Python APIs can authenticate with google using the ``GOOGLE_APPLICATION_CREDENTIALS`` environmental variable `(See Google authentication details) <https://cloud.google.com/sdk/docs/authorizing>`_.
+Authentication obtained via ``gcloud auth login`` does not work well with secrets management and is not used by many APIs;
+to use user credentials for API calls, you should also also generate credentials for this with ``gcloud auth application-default login``.
 
-* If gcloud is a fresh install, initialize and grab a keyfile::
+* If gcloud is a fresh install,
 
     > gcloud init
```

Comment:
These seem to be interpreted as "quotes" by Githubs auto rendering.  I'd replace with a ~ or $.